beta/do-app-platform-op-cli/README.md

```diff
@@ -28,31 +37,32 @@ Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
   > **Note**
   >
   > If you don't have a DigitalOcean account, you can sign up for a free trial with starting credit: <https://try.digitalocean.com/freetrialoffer/>
-- A Mac or Linux terminal with Bash, Zsh, or Fish
+- Access to a terminal on Mac or Linux Bash, Zsh, or on Windows using Powershell
 
 ## Getting started
 
 ### Step 1: Install 1Password and DigitalOcean tools
 
-Install the following on your Mac or Linux machine:
+Install the following on your Mac, Linux or Windows machine:
```

Comment:
Order in style guide order (Mac, Windows, Linux)

Reply:
```suggestion
Install the following on your Mac, Windows or Linux machine:
```